### ISSUE TITLE
Chrome: Uncaught ReferenceError: asciinema is not defined

### ISSUE THREAD

**Issue by tvannahl** [asciinema/asciinema-player#39]
I tried the basic example with the current version and ran it in Firefox (49) as well as in Google Chrome (54.0.2840.90). While it worked in Firefox it did not in Chrome.

I got the error:
```
Uncaught ReferenceError: asciinema is not defined
    at HTMLElement.AsciinemaPlayerProto.createdCallback (file:///…/asciinema-player.js:83:5)
    at file:///…/asciinema-player.js:93:12
    at file:///…/asciinema-player.js:94:3
```

Where I used the following *test.html*:
```html
<html>
<head>
  <link rel="stylesheet" type="text/css" href="asciinema-player.css" />
</head>
<body>
  <asciinema-player src="tty-records/foo.rec"></asciinema-player>
  <script src="asciinema-player.js"></script>
</body>
</html>
```

**Comment by sickill** [comment]
@tvannahl does the player on https://asciinema.org/ work for you in Chrome?

**Comment by sickill** [comment]
Just released v2.3.1 which fixes this problem: https://github.com/asciinema/asciinema-player/releases/tag/v2.3.1
